experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml

```diff
@@ -0,0 +1,188 @@
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - /mnt/parameters.yml
+    - /experiments/openebs/openebs_components.yml
+    - vars.yml
+  
+  tasks:
+    - block:
+
+      - include_tasks: /utils/runtime/create_testname.yml  
+
+      - include_tasks: openebs_pool_disk_loss_ansible_prerequisites.yml
+
+      # GENERATE EXPERIMENT RESULT NAME AND CREATE CHAOS RESULT RESOURCE
+      - name: Construct chaos result name (experiment_name)
+        set_fact:
+          c_experiment: "{{ lookup('env','CHAOSENGINE') }}-{{ c_experiment }}"
```

Comment:
Can you please change this task according to other tasks, pls refer https://github.com/litmuschaos/litmus/blob/master/experiments/generic/pod_delete/pod_delete_ansible_logic.yml

Reply:
Done